databuilder/databuilder/task/search/search_data_queries.py

```diff
@@ -11,82 +11,76 @@
     MATCH (db:Database)<-[:CLUSTER_OF]-(cluster:Cluster)
     <-[:SCHEMA_OF]-(schema:Schema)<-[:TABLE_OF]-(table:Table)
     {publish_tag_filter}
-    OPTIONAL MATCH (table)-[:DESCRIPTION]->(table_description:Description)
     OPTIONAL MATCH (schema)-[:DESCRIPTION]->(schema_description:Description)
-    OPTIONAL MATCH (table)-[:DESCRIPTION]->(prog_descs:Programmatic_Description)
-    WITH db, cluster, schema, schema_description, table, table_description,
-    COLLECT(prog_descs.description) as programmatic_descriptions
+    OPTIONAL MATCH (table)-[:DESCRIPTION]->(table_description:Description)
     OPTIONAL MATCH (table)-[:TAGGED_BY]->(tags:Tag) WHERE tags.tag_type='default'
-    WITH db, cluster, schema, schema_description, table, table_description, programmatic_descriptions,
-    COLLECT(DISTINCT tags.key) as tags
+    WITH db, cluster, schema, schema_description, table, table_description,
+    COLLECT(DISTINCT toLower(tags.key)) as tags
     OPTIONAL MATCH (table)-[:HAS_BADGE]->(badges:Badge)
-    WITH db, cluster, schema, schema_description, table, table_description, programmatic_descriptions, tags,
-    COLLECT(DISTINCT badges.key) as badges
-    OPTIONAL MATCH (table)-[read:READ_BY]->(user:User)
-    WITH db, cluster, schema, schema_description, table, table_description, programmatic_descriptions, tags, badges
-    OPTIONAL MATCH (table)-[:COLUMN]->(col:Column)
+    WITH db, cluster, schema, schema_description, table, table_description, tags,
+    COLLECT(DISTINCT toLower(badges.key)) as badges
+    MATCH (table)-[:COLUMN]->(col:Column)
     OPTIONAL MATCH (col)-[:DESCRIPTION]->(col_description:Description)
     WITH db, cluster, schema, schema_description, table, table_description, tags, badges,
-    programmatic_descriptions,
-    COLLECT(col.name) AS columns, COLLECT(col_description.description) AS column_descriptions
+    COLLECT(toLower(col.name)) AS columns, COLLECT(col_description.description) AS column_descriptions
     OPTIONAL MATCH (table)-[:LAST_UPDATED_AT]->(time_stamp:Timestamp)
     {additional_field_match}
-    RETURN db.name as database, cluster.name AS cluster, schema.name AS schema,
+    RETURN toLower(db.name) as database, cluster.name AS cluster, toLower(schema.name) AS schema,
     schema_description.description AS schema_description,
-    table.name AS name, table.key AS key, table_description.description AS description,
+    toLower(table.name) AS name, table.key AS key, table_description.description AS description,
     time_stamp.last_updated_timestamp AS last_updated_timestamp,
     {{
         {usage_fields}
     }} AS usage,
     columns,
     column_descriptions,
     tags,
-    badges,
     {additional_field_return}
-    programmatic_descriptions
+    badges
```

Comment:
why move this

Reply:
because we need a comma before the additional fields and it felt weird to make people put a comma in there

Reply:
oh right order doesn't matter in `to_document`